src/extension/index.activateGithubAnalyses.ts

```diff
@@ -110,13 +123,23 @@ export function activateGithubAnalyses(disposables: Disposable[], store: Store,
         }
 
         const [, user, repoName] = (() => {
-            // Example: https://github.com/user/repoName.git
             const matchHTTPS = origin.match(/https:\/\/github\.com\/([^/]+)\/([^/]+)/);
             if (matchHTTPS) return matchHTTPS;
 
-            // Example: git@github.com:user/repoName.git
-            const matchSSH = origin.match(/git@github\.com:([^/]+)\/([^/]+)/);
-            if (matchSSH) return matchSSH;
+            // Try githubCodeScanningEndpoint first, then try it without a "api." prefix.
+            // This is to support GitHub Enterprise Server, which may have a different endpoint.
```

Comment:
Can you give an example of what a GHES URL might look like? That would help for future readers who aren't as versed in Enterprise instances,